### ISSUE TITLE
Adding Activation layer

### ISSUE THREAD

**Issue by asingh9530** [keras-team/keras-core#772]
Hi @fchollet,

Wanted to know we planning to add more activation layers as currently only `relu` and its successors  along with `softmax` are added ? if yes I can add most of then which are in ops in a single PR.
